dash/examples/ex.06.pattern-block-visualizer/main.cpp

```diff
@@ -240,11 +301,16 @@ cli_params parse_args(int argc, char * argv[])
     } else if (flag == "-p") {
       params.cout    = true;
       i -= 2;
+    } else {
+      print_usage(argv);
+      exit(EXIT_FAILURE);
     }
   }
+
   return params;
 }
 
+static
```

Comment:
I'd put it the other way around: there is really no sense in exposing anything but `main` in `main.cpp` here so there is no reason not have them `static`.  I think it's good practice to only expose those functions that actually need external linkage.